chart_review/commands/accuracy.py

```diff
@@ -4,12 +4,20 @@
 import os
 
 import rich
+import rich.box
 import rich.table
+import rich.text
 
 from chart_review import agree, cli_utils, cohort, common, config, console_utils
 
 
-def accuracy(reader: cohort.CohortReader, truth: str, annotator: str, save: bool = False) -> None:
+def accuracy(
+    reader: cohort.CohortReader,
+    truth: str,
+    annotator: str,
+    save: bool = False,
+    verbose: bool = False,
```

Comment:
nit: is this the only way we're going to want to make this more verbose? I suppose you can always change it later